libraries/GCS_MAVLink/examples/routing/routing.cpp

```diff
@@ -122,5 +123,10 @@ void loop(void)
     hal.scheduler->delay(1000);
 }
 
+/* dummy methods to avoid having to link against AP_Camera */
```

Comment:
Agreed.  Went through this a bit with MdB yesterday.  We've had a rule against things like `CAMERA_DISABLED` in library code (quite reasonably) - except now we have "vehicle code" which is common in the `libraries` directory.  But in the absence of smart compilers...